src/h5web/providers/jupyter/models.ts

```diff
@@ -1,7 +1,6 @@
-import type { EntityKind } from '../models';
+import type { Attribute, EntityKind } from '../models';
 
 export interface JupyterMetaResponse {
-  id: string;
```

Comment:
`id` was removed in the dev API. As we do not make use of it, we might as well remove it from the stable API interface.